README.md

```diff
@@ -39,6 +39,11 @@ cd CubeCobra/..
 ln -s CubeCobra/cubecobrasecrets.example cubecobrasecrets
 ```
 
+In a separate terminal window, make sure your MongoDB database instance is up and running
```

Comment:
Convert MongoDB to a link or add a link to install instructions?
https://docs.mongodb.com/manual/installation/

Reply:
This isn't an instruction to install mongoDB, it's an instruction to start an instance of it. Running seed.js will fail unless there is an instance of mongod running (since that means the database isn't up)